webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -271,7 +272,7 @@ class SubscriptionsPage extends Component {
             />
 
             <div id="subscriptions-table" className="modal-container">
-              {SCAAlert}
+              {isManifestImported && SCAAlert}
```

Comment:
you can just use this as defined in angular?  how does that work?

Reply:
It just happens to be named the same ;) 

going from Angular archaeology to React archaeology..

It's a Redux selector used here -> https://github.com/jeremylenz/katello/blob/0e11585c3859472a2decc4e6579cda25f9753855/webpack/scenes/Subscriptions/index.js#L38-L39

Reply:
It was very convenient that we were already using it on this page, so that code was already there

Reply:
ahahha makes sense!